### ISSUE TITLE
fetch in macro has CORS error

### ISSUE THREAD

**Issue by Baton34** [ONLYOFFICE/onlyoffice.github.io#332]
I try to fetch some webApi queries to my organization server in macros, but fetch stopped with error 
> Access to XMLHttpRequest at 'https://google.com/' from origin 'file://' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

For example I used google.com.

`(function()
{
    fetch('https://google.com')
  .then(response => console.log(response));
})();`

How to do right fetch?

**Comment by Rita-Bubnova** [comment]
Sorry for the long wait.
Is this issue relevant?

**Comment by Baton34** [comment]
Yes, and for now editor falling to error with "Use 'Save as'  option to save file" popup (translated from russian).

**Comment by Baton34** [comment]
@regent112 question about CORS, not async method.
What output do you see in your console? I'm getting "1 2".
(function()
{
    asyncCall();
})();

function resolveAfter2Seconds() {
  return new Promise(() => {
    setTimeout(() => {
      console.log("2");
    }, 2000);
  });
}

async function asyncCall() {
  resolveAfter2Seconds();
  console.log("1");
}
